### DISCUSSION TITLE
PQ-Splitter does not work anymore?

### DISCUSSION THREAD

**Discussion by LettaLet** [2Abendsegler/GClh#2199] [Questions] [answered]
Hi,
It seems that PQ-Splitter does not work anymore. Are there other users with this problem too? I'm using chrome with pop-up blocker disabled.

**Comment by capoaira** [comment, marked as answer]
Yes, we had some problems with the PQ-Splitter. Please test temporary fix https://github.com/2Abendsegler/GClh/blob/1b89237ef1930298c82896ecb52c9711e85cc515/gc_little_helper_II.user.js?raw=true Does it fix the problem?
Link to the current version: https://github.com/2Abendsegler/GClh/blob/master/gc_little_helper_II.user.js?raw=true
We will roll out the new version soon. 

Related PR #2187

**Reply by LettaLet** [reply]
The workaround fixed it.
Thanks.
